README.md

```diff
@@ -1,17 +1,34 @@
-# Fluentd image to send Kuberntes logs to CloudWatch
+# Fluentd image to send Kubernetes logs to CloudWatch
 
-## Quickstart
+## Deployment
 
 1. Create AWS IAM user which has permission to store logs to CloudWatch
 
-2. Clone this repository
+2. Set environment variables for the AWS IAM user including region
 
-3. You may want to edit the ConfigMap to change the region and log group name
-
-4. Run:
 ```
-echo -n "accesskeyhere" > aws_access_key
-echo -n "secretkeyhere" > aws_secret_key
-kubectl create secret --namespace=kube-system generic fluentd-secrets --from-file=aws_access_key --from-file=aws_secret_key
-kubectl apply -f fluentd-cloudwatch-daemonset.yaml
+export AWS_ACCESS_KEY_ID=<your key>
+export AWS_SECRET_ACCESS_KEY=<your secret>
+export AWS_DEFAULT_REGION=<your region>
```

Comment:
This is a bit risky as there's unfortunately a bit high chance that the user could just skip this step and end up using their currently loaded credentials (which likely have quite broad permissions). It would be better to name the variables differently from AWS's default credentials and also possibly suggest passing the variables directly to the deploy script, ex.
```
LOGGING_AWS_ACCESS_KEY_ID=foobar LOGGING_AWS_SECRET_ACCESS_KEY=secretfoo  LOGGING_AWS_DEFAULT_REGION=bar ./deploy.sh 
```
This would prevent the variables from be exposed to other processes that are launched later, would not overwrite the currently loaded credentials and generally user only needs to run the script once.